docs/troubleshooting-guide/loggingin.md

```diff
@@ -22,9 +22,7 @@ Remove the old token from the Google Auth SSO tab in the Admin Panel and create
 ## Helpful tidbits
 
 ### Accessing the Metabase server logs
-While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can also access them from the drop down menu in the upper right hand corner.
-
-![Logs Screenshot](images/ServerLogs.png)
+While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, if you are logged into Metabase with an admin account, you can also access them from the gear drop down menu in the upper right hand corner -> Admin -> Troubleshooting -> Logs.
```

Comment:
I'd revise this to:

`While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can view the server logs as well as task and job logs in the Admin Panel by clicking the Troubleshooting tab in the nav bar.`